src/Http/Controllers/Slack/SlackUserOAuthController.php

```diff
@@ -86,9 +94,21 @@ public function callback()
                 ->with('error', 'An error occurred while exchanging credentials with slack. ' . $e->getMessage());
         }
 
-        return redirect()->route('seatnotifications.index')
-            ->with('success', 'Now we know who to notify and you can subscribe to any of the available notifications.');
+        $driver = request()->session()->pull('herpaderp.seatnotifications.subscribe.driver');
+        $notification = request()->session()->pull('herpaderp.seatnotifications.subscribe.notification');
+        $characters_filter = request()->session()->pull('herpaderp.seatnotifications.subscribe.characters_filter');
```

Comment:
The driver should only be annoyed with stuff concerning the way binding and notification are sent.

We should find a way to extract filters from this logic

Reply:
you are right, i copy that from discord auth part. If i read this correctly you had introduced it probably its obsolete:
https://github.com/herpaderpaldent/seat-notifications/blame/23269a36fd8ac8f2f4be9e7fcae6b989bcfd8cbb/src/Http/Controllers/Discord/DiscordOAuthController.php#L101-L102

Reply:
I didn't say obsolete but that it was a duplicated area ;)